src/mgr/DaemonHealthMetricCollector.h

```diff
@@ -12,7 +12,7 @@ class DaemonHealthMetricCollector {
   static std::unique_ptr<DaemonHealthMetricCollector> create(daemon_metric m);
   void update(const DaemonKey& daemon, const DaemonHealthMetric& metric) {
     if (_is_relevant(metric.get_type())) {
-      reported = _update(daemon, metric);
+      if (_update(daemon, metric)) reported = true;
```

Comment:
ar, can you put that into a new line?

Reply:
I pushed it again but I don't know if I understood you correctly.